lighthouse-core/formatters/partials/critical-request-chains.html

```diff
@@ -1,62 +1,3 @@
-<style>
```

Comment:
Done. I just feel that having random styles in many places isn't good. But, yeah, it was unrelated change to this PR.